Formula/libxml2.rb

```diff
@@ -57,12 +57,10 @@ def install
     ENV.deparallelize
     system "make", "install"
 
-    if build.with? "python"
-      cd "python" do
-        # We need to insert our include dir first
-        inreplace "setup.py", "includes_dir = [", "includes_dir = ['#{include}', '#{MacOS.sdk_path}/usr/include',"
-        system "python", "setup.py", "install", "--prefix=#{prefix}"
-      end
+    cd "python" do
+      # We need to insert our include dir first
+      inreplace "setup.py", "includes_dir = [", "includes_dir = ['#{include}', '#{MacOS.sdk_path}/usr/include',"
```

Comment:
It's not, by a long way, but has been this way forever. Happy to fix if you like.